guides/common/modules/proc_upgrading-the-external-database-operating-system.adoc

```diff
@@ -0,0 +1,24 @@
+[id="Upgrading_the_External_Database_Operating_System{context}"]
+= Upgrading the external database operating system
+
+You can upgrade an external database from {EL} 8 to {EL} 9 while upgrading {Project} from {ProjectVersionPrevious} to {ProjectVersion}.
```

Comment:
```suggestion
If your {Project} uses an external database, you can upgrade the database from {EL} 8 to {EL} 9 while upgrading {Project} from {ProjectVersionPrevious} to {ProjectVersion}.
```
Does this make sense? I'm looking at the preview and think it might be helpful to give users a hint on whether the procedure is relevant for their deployment.